scripts/run-migrations.ts

```diff
@@ -118,13 +125,41 @@ class MigrationRunner {
     
     execSync(`rm -rf ${tempMigrationsDir}`, { cwd: process.cwd() });
   }
+
+  private captureFilteredOutput(): void {
```

Comment:
Instead of capturing the filtered output, we are missing the important part where we post the output as a Github comment on the PR